util/Migrator/MigratorConstants.cs

```diff
@@ -0,0 +1,8 @@
+namespace Bit.Migrator;
```

Comment:
❌ with lint:

> /MigratorConstants.cs(1,1): error CHARSET: Fix file encoding.

Make sure this is committed as UTF-8.

Reply:
Fixed.